week-7/schedule.md

```diff
@@ -7,28 +7,25 @@
 | 09:45   | Check-in                       |                           |
 | 10:00   | Intro presentation             |                           |
 | 10:30   | [Express intro][express-intro] | Express                   |
-| 11:00   | Learn Express (tbc)            | Routing, Params           |
+| 11:00   | [Learn Express][learn-express] | Routing, Params           |
 | _13:00_ | _Lunch_                        |                           |
 | 14:00   | REST APIs (tbc)                | REST, Building a JSON API |
-| 16:00   | Tech for Better                |                           |
+| 16:00   | Technical spikes               |                           |
 
-[express-intro]: tbc
+[express-intro]: https://github.com/oliverjam/express-intro
+[express-intro]: https://github.com/oliverjam/learn-express
```

Comment:
Reference (on the left) to be updated to [learn-express]

Reply:
whoops